csrankings.csv

```diff
@@ -757,7 +757,8 @@ Alin Dobra,University of Florida,http://www.cise.ufl.edu/~adobra,smEOOBsAAAAJ
 Alin Olimpiu Albu-Schäffer,TU Munich,https://www.professoren.tum.de/en/albu-schaeffer-alin,yP_zL3gAAAAJ
 Alina Ene,Boston University,http://www.bu.edu/cs/alina-ene-2,v1_kfR8AAAAJ
 Alina Oprea,Northeastern University,http://www.ccs.neu.edu/home/alina,16J3izoAAAAJ
-Alina Sirbu,University of Pisa,http://kdd.isti.cnr.it/people/s%C3%AErbu-alina,o-G11bQAAAAJ
+Alina Sirbu,University of Pisa,http://kdd.isti.cnr.it/people/s%C3%AErbu-alina,o-G11bQAAAAJ
+Alina Zare,University of Florida,https://faculty.eng.ufl.edu/machine-learning/,OviqAJsAAAAJ

```

Comment:
Not in computer science.
https://www.cise.ufl.edu/people/faculty/

Reply:
Alina Zare has graduate faculty status in the computer science department.

Reply:
Can you point to evidence of this?

Reply:
Sorry, I don't see proof online, but I know first-hand.

Reply:
That's not enough, sorry. The most expedient way to handle this is to have the faculty member indicate this publicly on this thread.